assets/scss/mq.scss

```diff
@@ -2,14 +2,20 @@
 
 $mq-responsive: true;
 
+$small: 320px;
```

Comment:
[Here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media), in the  Accessibility concerns section, it says that both `em` and `px` are valid units, but `em` works better if the user changes the browser text size. Moreover, we are removing all `px` from the web, I think we should be consistent with that

Reply:
Makes sense and I'd want to involve the design team for translating [Carbon breakpoints](https://www.carbondesignsystem.com/guidelines/2x-grid/basics#breakpoints) to the correct values in `em` (issue #600). Right now, I preffer leaving them as they are.

Reply:
I left you a comment [in the issue](https://github.com/Qiskit/qiskit.org/issues/600#issuecomment-627817733) because we can get the `em` values from the information we have

Reply:
Done in this PR.